src/v/cluster/metadata_dissemination_service.cc

```diff
@@ -317,7 +317,11 @@ ss::future<> metadata_dissemination_service::dispatch_disseminate_leadership() {
                  return dispatch_one_update(br_update.first, br_update.second);
              })
       .then([this] { cleanup_finished_updates(); })
-      .finally([this] { _dispatch_timer.arm(_dissemination_interval); });
+      .finally([this] {
```

Comment:
```
c/dissemination: do not arm dispatch timer when gate is closed
```

is this fixing something? it looks ok: the timer is cancelled in stop() before the gate is closed. because of that it looks like the patch is fixing a bug, but there isn't an indication of any issue being fixed.

Reply:
i will just skip this commit, will add it when we will be refactoring the dissemination service. this is merely an optimization.